From 16969a4451e09d1bd0b7aff2208d514a6bcbf011 Mon Sep 17 00:00:00 2001
From: Stijn Segers <foss@volatilesystems.org>
Date: Tue, 20 Sep 2022 23:05:43 +0200
Subject: [PATCH 08/18] mvebu: rb5009: use QCA8081 label for 2.5 GbE port in
 DTS

The SFP+ interface is clearly labeled as such in the DTS,
unlike the 2,5 GbE QCA8081. Label it to distinguish it from
the GbE ports as well. Taken from robimarko's tree at
https://github.com/robimarko/openwrt/tree/rb5009-5.15.
---
 .../files/arch/arm64/boot/dts/marvell/armada-7040-rb5009.dts  | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/target/linux/mvebu/files/arch/arm64/boot/dts/marvell/armada-7040-rb5009.dts b/target/linux/mvebu/files/arch/arm64/boot/dts/marvell/armada-7040-rb5009.dts
index 0babac6f08..0c9742d2d8 100644
--- a/target/linux/mvebu/files/arch/arm64/boot/dts/marvell/armada-7040-rb5009.dts
+++ b/target/linux/mvebu/files/arch/arm64/boot/dts/marvell/armada-7040-rb5009.dts
@@ -301,7 +301,7 @@
 				reg = <9>;
 				label = "p1";
 				phy-mode = "sgmii";
-				phy-handle = <&switch0phy9>;
+				phy-handle = <&qca8081>;
 				managed = "in-band-status";
 				nvmem-cells = <&macaddr_hard>;
 				nvmem-cell-names = "mac-address";
@@ -358,7 +358,7 @@
 			#address-cells = <1>;
 			#size-cells = <0>;
 
-			switch0phy9: switch0phy9@0 {
+			qca8081: qca8081@0 {
 				reg = <0>;
 			};
 		};
-- 
2.42.1

